doc/source/whatsnew/v0.17.1.txt

```diff
@@ -60,6 +60,8 @@ Performance Improvements
 
 - Release the GIL on most datetime field operations (e.g. ``DatetimeIndex.year``, ``Series.dt.year``), normalization, and conversion to and from ``Period``, ``DatetimeIndex.to_period`` and ``PeriodIndex.to_timestamp`` (:issue:`11263`)
 - Release the GIL on some srolling algos (``rolling_median``, ``rolling_mean``, ``rolling_max``, ``rolling_min``, ``rolling_var``, ``rolling_kurt``, `rolling_skew`` (:issue:`11450`)
+- Release the GIL when reading a file using the TextReader class(``read_csv``, ``read_table``).
+  The GIL is now released around the tokenizer functions and the conversion functions(_string_convert excluded).
```

Comment:
just say: ``Release the GIL when reading and parsing text files in ``pd.read_csv``, (:issue:`12272`)``